lib/jsdom.js

```diff
@@ -361,6 +364,12 @@ function getConfigFromArguments(args, callback) {
     config.url = toFileUrl(config.file);
   }
 
+  if (config.url) {
+    // backslashes were originally not allowed in URLs, so this is error-handling only
+    // comparable to chrome which converts all backslashes to foward slashes
+    config.url = config.url.replace(/\\/g, '/');
```

Comment:
Hmm shouldn't we just do config.url = url.resolve("about:blank", config.url)` which would normalize everything for us?